recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
 
 test:
     requires:
-        - python {{ environ['PY_VER'] + '*' }}  # [win]
+        - python {{ PY_VER ~ '*' }}  # [win]
```

Comment:
The bug in the latest `conda-build` that prevent us from using  `environ['PY_VER']` as that is recognized as another selector.